### PR TITLE
Updated installer to pickup changes to Agent module configuration.

### PR DESCRIPTION
**Pull request by asridharan**
These changes allow the Agent to run on the Master and selectively
configure docker and mesos bridges.

### CODE REVIEW

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -96,9 +96,27 @@ package:
...
         "type": "bridge"
       }
-  - path: /etc/dcos/network/overlay
+  - path: /etc/dcos/network/overlay-master
     content: |
       {{ dcos_overlay_network }}
+  - path: /etc/dcos/network/overlay-agent-master
+    content: |
+      {
+        "master": "zk://leader.mesos:2181/mesos",
```
Comment (sargun) [inline gen/dcos-config.yaml]: Should this be leader.mesos?

File: gen/dcos-config.yaml [hunk 2]
```diff
@@ -96,9 +96,27 @@ package:
...
+        {
+          "allocate_subnet": true,
+          "mesos_bridge": false,
+          "docker_bridge": true
+        }
+      }
+  - path: /etc/dcos/network/overlay-agent
+    content: |
+      {
+        "master": "zk://leader.mesos:2181/mesos",
```
Comment (sargun) [inline gen/dcos-config.yaml]: Should this be leader.mesos?

File: gen/dcos-config.yaml (hunk 1 above)
Comment (asridharan) [inline gen/dcos-config.yaml]: Only the leading Master can allocate the subnet, vtep IP and MAC, so the Agent (even when running on the Master) will always talk to the leading master.

Ofcourse, in the process of implementing the replicated log for the modules, so we don't currently support a multi master setup. Will do by end of this week.

Comment (asridharan) [inline gen/dcos-config.yaml]: Oh , whats the URI for the local zk ? I think need to point it to the local zk instead of the leader.

**Pull request merged**